float32 yaw
float32 pitch
float32 dis
bool is_aimed
bool is_target_exists
